src/main/java/uk/gov/hmcts/ccd/domain/service/createcase/SubmitCaseTransaction.java

```diff
@@ -121,6 +130,12 @@ public CaseDetails submitCase(Event event,
             caseDetailsAfterCallback
         );
 
+        if (this.applicationParams.getCaseGroupAccessFilteringEnabled()) {
+            CaseAccessGroupUtils caseGroupAccessTypeUtils = new CaseAccessGroupUtils();
```

Comment:
We should be injecting the `CaseAccessGroupUtils` rather than creating a new instance here each time.

Reply:
fixed in #2389